app/src/main/java/com/asfoundation/wallet/repository/SharedPreferencesRepository.kt

```diff
@@ -150,6 +150,8 @@ class SharedPreferencesRepository(context: Context) : PreferencesRepositoryType
         .apply()
   }
 
+  override fun getGamificationLevel() = pref.getInt(GAMIFICATION_LEVEL, -1)
```

Comment:
we already have a method to get this, `getLastShownLevel`

Reply:
The logic here is slightly different, even if it represents the same concept. Since we get this information from the server at some points already, I'm using these methods to make sure that information is independent from what was shown or not. This guarantees maximum trust in that value without having to make extra server calls at app start.